### PR TITLE
[APM-CI] windows build

### PR DESCRIPTION
**Pull request by kuisathaverat**
* New windows build process
* Possibility to trigger the job with a comment
* Fixed lib versions

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -121,3 +121,11 @@ resharper_csharp_braces_for_fixed=required_for_multiline
 resharper_csharp_braces_for_ifelse=required_for_multiline
 
 resharper_csharp_accessor_owner_body=expression_body
+
+[Jenkinsfile]
```
Comment (gregkalapos) [inline .editorconfig]: @kuisathaverat jfyi: we try to keep the `.editorconfig` files in sync across .NET repos within Elastic. @elastic/dotnet Are we ok with this change? Do you want to sync this to other repos? I think it's ok to have this...

Comment (Mpdreamz) [reply .editorconfig]: Lgtm :+1:

Comment (kuisathaverat) [reply .editorconfig]: I do not have a strong opinion, as you preference

Comment (gregkalapos) [reply .editorconfig]: Great, then let's push it as it is now. We are ok with this. 👍

**Pull request merged**